internal/database/database_test.go

```diff
@@ -13,30 +13,50 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package database
+package database_test
```

Comment:
Is this name intentional? Normally tests reside in the same package as the code under test.

Reply:
Tests can either be in the same package or as a "*_test" package.
This change is because the "test" package imports the "database" package and that circular dependency is not allowed.